src/plugins/expression/expression-tile.tsx

```diff
@@ -65,6 +71,13 @@ export const ExpressionToolComponent: React.FC<ITileProps> = observer((props) =>
     onUnregisterTileApi
   });
 
+  const mathfieldAttributes = {
+    ref: readOnly ? readOnlyMf : mf,
+    value: content.latexStr,
+    onInput: !readOnly ? handleChange : () => null,
```

Comment:
Does it work if this is:
```suggestion
    onInput: !readOnly ? handleChange : undefined,
```